### PR TITLE
docs: add AWS Plugin to plugin directory

### PR DESCRIPTION
**Pull request by greenpau**
Add a workflow executor plugin for AWS services.

### Motivation

Provide ability to trigger the execution of AWS services.

### Modifications

No changes.

### Verification

N/A

### CODE REVIEW

File: docs/plugin-directory.md [hunk 1]
```diff
@@ -14,3 +14,4 @@
 | [Chaos Mesh Plugin](https://github.com/xlgao-zju/argo-chaos-mesh-plugin) | Run Chaos Mesh experiment |
 | [Pull Request Build Status](https://github.com/LinuxSuRen/gogit) | Send build status of pull request to Git provider |
 | [Atomic Workflow Plugin](https://github.com/LinuxSuRen/argo-workflow-atomic-plugin) | Stop the workflows which comes from the same `WorkflowTemplate` and have the same parameters |
+| [AWS Plugin](https://github.com/greenpau/argo-workflows-aws-plugin) | Argo Workflows Executor Plugin for AWS Services, e.g. SageMaker Pipelines, Glue, etc. |
```
Comment (agilgur5) [inline docs/plugin-directory.md:17]: you'll need to add `SageMaker` to `.spelling`

Comment (greenpau) [reply docs/plugin-directory.md:17]: @agilgur5 , 👍 done!

**Review by agilgur5 — COMMENTED** [review]
DCO is [missing](https://github.com/argoproj/argo-workflows/pull/12169/checks?check_run_id=18507752560) in your second commit

**Comment by agilgur5** [comment]
Replaced by #12171

**Pull request closed without merging**